public/js/reducers/pause.js

```diff
@@ -107,35 +107,35 @@ function update(state = initialState, action, emit) {
   return state;
```

Comment:
I don't think this file has the `// @flow` at the top of it so I don't think flow was looking in here.